.github/workflows/integration.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       - name: Prepare environment
         run: |
           pip install typer pyyaml gitpython packaging
-          git clone https://github.com/DIRACGrid/DIRAC.git -b "${{ matrix.dirac-branch }}" /tmp/DIRACRepo
+          git clone https://github.com/chaen/DIRAC.git -b "diracx_sandbox" /tmp/DIRACRepo
```

Comment:
@chaen This is also ready for review modulo this change which will be reverted before merging

Reply:
:+1:  I've added just the 2 minor fixes we needed (location for write_credentials and DiracClient taking a Pref object)